### PR TITLE
Label updates again

### PR DESCRIPTION
**Pull request by caffodian**
see #69 for the list

### CODE REVIEW

File: structuredheadings/plugin.js [hunk 1]
```diff
@@ -301,22 +317,23 @@
...
       editor.on("dataReady", function () {
         self.currentScheme = self.detectScheme();
       });
 
       //Format Dropdown
       editor.ui.addRichCombo("NumFormats", {
-        label: "Formats",
-        title: "Numbering Formats",
+        label: "Formatting",
+        title: "Text Formatting",
```
Comment (winhamwr) [inline structuredheadings/plugin.js:329]: [this](https://trello.com/c/b8fGNlrA/37-editor-structured-headings-numbering-1-1-1-1-1-1-new-customers#comment-58ffb2786289aeba5a336e99) specifies "Numbering" for this menu. Where did the "Text Formatting" label idea come from?

Comment (winhamwr) [reply structuredheadings/plugin.js:329]: (or am I confused on what this label represents?)

Comment (caffodian) [reply structuredheadings/plugin.js:329]: Text formatting is the left one (p, h2-h5, etc) not the right one 

this was in https://docs.google.com/document/d/1KOKwdgYA8_bzHAnqCx1jhzP-v7SlM5d7vj_-I0yiAYQ/edit

File: structuredheadings/plugin.js [hunk 2]
```diff
@@ -394,15 +413,15 @@
 
       //Style Dropdown
       editor.ui.addRichCombo("NumStyles", {
-        label: "Styling",
-        title: "Styling",
+        label: "Numbering",
+        title: "Numbering",
```
Comment (caffodian) [inline structuredheadings/plugin.js:417]: this is the rename you're looking for (the right dropdown)

File: structuredheadings/plugin.js (hunk 1 above)
Comment (winhamwr) [reply structuredheadings/plugin.js:329]: Are we referring to:

> The tooltip on rollover of the format dropdown says “Numbering Formats”. It should say “Text Formatting”

I don't think that's Jana suggesting that the dropdown be changed to "Text Formatting", just that the tooltip shouldn't say "Numbering Formats" when it's not the numbering section.

I don't see how "Text Formatting" is an improvement over "Formatting".

Comment (caffodian) [reply structuredheadings/plugin.js:329]: The title gets added only as a title attribute, so it doesn't actually display anywhere except for mouseover (I think it should be consistent with the aria label as well, though my accessibility knowledge is fuzzy and lacking in this area)  

The Formatting part is to handle the default state (only really shows when you are in a non-paragraph/h block, such as a new table cell)

File: structuredheadings/plugin.js [hunk 3]
```diff
@@ -173,6 +183,9 @@
       ]
     };
 
+    editor.config.autonumberDefaultScheme =
+    editor.config.autonumberDefaultScheme || "1.1.1.1.1.";
```
Comment (bocribbz) [inline structuredheadings/plugin.js]: can this be more inner indented to make it obvious is the value of the above assignment?

File: structuredheadings/plugin.js (hunk 2 above)
Comment (bocribbz) [reply structuredheadings/plugin.js:417]: wooohooo

**Comment by bocribbz** [comment]
This looks ok to be, but for that small indentation if its possible.

**Review by bocribbz — APPROVED** [review]

**Pull request merged**